RELEASE_GUIDE.md

```diff
@@ -6,6 +6,13 @@ Lens releases are built by CICD automatically on git tags. The typical release p
 
 1. If doing a patch release checkout the `release/vMAJOR.MINOR` branch for the appropriate `MAJOR`/`MINOR` version and manually `cherry-pick` the PRs required for the patch that were commited to master. If there are any conflicts they must be resolved manually. If necessary, get assistance from the PR authors.
 
+This can be helped (if you have the `gh` CLI installed) by running:
```

Comment:
I guess you could mention needing `jq` here too, since it is used below

Reply:
Maybe in a future revision.